tools/vigiechiro/vigiechiro_macros.xml

```diff
@@ -29,12 +29,12 @@
         </citations>
     </xml>
     <xml name="vigiechiro_bilanPF_bilanRP_requirements">
-        <requirements>
-            <requirement type="package" version="1.4.3">r-reshape2</requirement> 
-            <requirement type="package" version="1.11.4">r-data.table</requirement>
-            <requirement type="package" version="0.4">r-dt</requirement>
-            <requirement type="package" version="1.2">r-htmlwidgets</requirement>
-            <requirement type="package" version="2.6">pandoc</requirement>
-        </requirements>	
+        <requirement type="package" version="1.4.3">r-reshape2</requirement> 
```

Comment:
@Alanamosse do you know the yield feature or Galaxy macros?

https://planemo.readthedocs.io/en/latest/writing_advanced.html#parameterizing-xml-macros-with-yield

it looks like this can help here ...

Reply:
@bgruening Yes I know this feature but can't see how it helps here..

Reply:
.. got a flash ! maybe I do see something .. I try it !